frontends/web/src/components/transactions/components/tx-detail-dialog/tx-detail-dialog.tsx

```diff
@@ -0,0 +1,167 @@
+import { AddressOrTxId } from '@/components/transactions/components/tx-detail-dialog/address-or-tx-id';
+import styles from './tx-detail-dialog.module.css';
+
+type TProps = {
+  open: boolean;
+  onClose: () => void;
+  accountCode: string;
+  internalID: string;
+  note: string;
+  status: TTransactionStatus;
+  type: TTransactionType;
+  numConfirmations: number;
+  numConfirmationsComplete: number;
+  time: string | null;
+  amount: TAmountWithConversions;
+  explorerURL: string;
+};
+
+export const TxDetailsDialogNew = ({
+  open,
+  onClose,
+  accountCode,
+  internalID,
+  note,
+  status,
+  type,
+  numConfirmations,
+  numConfirmationsComplete,
+  time,
+  amount,
+  explorerURL,
+}: TProps) => {
+  const { t } = useTranslation();
+  const [transactionInfo, setTransactionInfo] = useState<TTransaction | null>(null);
+
+  useEffect(() => {
+    if (!transactionInfo && open) {
+      getTransaction(accountCode, internalID).then(transaction => {
+        if (!transaction) {
+          console.error(`Unable to retrieve transaction ${internalID}`);
+        }
+        setTransactionInfo(transaction);
+      }).catch(console.error);
+    }
+  }, [accountCode, internalID, open, transactionInfo]);
+
+  if (transactionInfo === null) {
+    return;
+  }
+
+  const sign = transactionInfo && getTxSign(transactionInfo.type);
+  const displayedSign = sign === '−' ? sign : '';
+
+  // Amount and Confirmations info are displayed using props data
+  // instead of transactionInfo because they are live updated.
+
+  return (
+    <Dialog
+      open={open && !!transactionInfo}
+      title={t('transaction.details.title')}
+      onClose={onClose}
+      slim
+      medium>
+      {transactionInfo && (
+        <div className={styles.container}>
+          <TxDetailHeader
+            status={status}
+            numConfirmations={numConfirmations}
+            numConfirmationsComplete={numConfirmationsComplete}
+            type={type}
+            amount={amount}
+            transactionInfo={transactionInfo}
+            time={time}
+            displayedSign={displayedSign}
+          />
+
+          <hr className={styles.separator} />
+
+          <Note
+            accountCode={accountCode}
+            internalID={internalID}
+            note={note}
+          />
+
+          {/* to or send address */}
+          {type && (
+            <TxDetailRow>
+              <p className={styles.label}>{type === 'send' ? t('send.confirm.to') : t('transaction.details.from')}</p>
+              <AddressOrTxId values={transactionInfo.addresses} />
+            </TxDetailRow>
+          )}
+
+          {/* fee */}
+          <TxDetailRow>
+            <p className={styles.label}>{t('transaction.fee')}</p>
+            <span>{displayedSign} <AmountWithUnit amount={transactionInfo.fee} unitClassName={styles.rowUnit} /></span>
+          </TxDetailRow>
+
+          {/* current fiat */}
+          <TxDetailRow>
+            <div className={styles.withTooltip}>
```

Comment:
is withTooltip needed?

Reply:
Thanks - no. Leftover as we removed the tooltip.